classes/services/PKPNavigationMenuService.inc.php

```diff
@@ -148,6 +152,13 @@ function getDisplayStatus(&$navigationMenuItem) {
 					$displayTitle = $templateMgr->fetch('frontend/components/navigationMenus/dashboardMenuItem.tpl');
 					$navigationMenuItem->setTitle($displayTitle, \AppLocale::getLocale());
 					break;
+				case NMI_TYPE_USER_PROFILE:
+					$templateMgr->assign('navigationMenuItem', $navigationMenuItem);
+					if ($this->_hasNMTreeNMIAssignmentWithChildOfNMIType($navigationMenu, $navigationMenuItem, NMI_TYPE_USER_DASHBOARD)) {
+						$displayTitle = $templateMgr->fetch('frontend/components/navigationMenus/dashboardMenuItem.tpl');
+						$navigationMenuItem->setTitle($displayTitle, \AppLocale::getLocale());
+					}
```

Comment:
If we're just accounting for reader users, I don't think we need to add the username and tasks. They won't have any tasks and we can leave out the logged in name.

Reply:
I removed that, and now the reader has a dashboard heading to the user/profile - It has the numbering beside it - is that a problem?

Reply:
Yeah, just check the user roles before the template is loaded, and add a boolean whether or not to display tasks (everyone sees it except readers).